apps/full-site-editing/bin/newspack-block-sync-phpcs.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<ruleset name="newspack-blocks">
+	<description>Detect issues with newspack blocks sync</description>
+
+	<rule ref="WordPress.WP.I18n">
+		<properties>
+		<property name="text_domain" type="array">
+			<element value="full-site-editing"/>
+		</property>
+		</properties>
+	</rule>
+
+	<file>../full-site-editing-plugin/newspack-blocks/synced-newspack-blocks</file>
```

Comment:
We are just testing synced newspack blocks in this scenario.

The normal phpcs command will catch any php issues with changed PHP files that are _tracked_ in the FSE plugin. I don't want to fail with the normal lint rules here, because newspack-blocks may uses different phpcs standards. (i.e. we should only add rules here which we know will break functionality on our end)